packages/typescript-estree/src/convert.ts

```diff
@@ -2342,6 +2362,7 @@ export class Converter {
             ? this.convertType(node.constraint)
             : undefined,
           default: node.default ? this.convertType(node.default) : undefined,
+          ...this.convertTypeParamVariances(node.modifiers),
```

Comment:
what do you think about changing it to `hasModifier` to be consistent with other use cases?


```suggestion
          in: hasModifier(SyntaxKind.InKeyword, node) || undefined,
          out: hasModifier(SyntaxKind.OutKeyword, node) || undefined,
```

Reply:
instead of `undefined` - could we make it `false`?
I'd love for us to standardise on not emitting `undefined` for properties